package.json

```diff
@@ -50,6 +50,7 @@
     "phantomjs-prebuilt": "^2.1.0"
   },
   "dependencies": {
+    "babel-polyfill": "^6.26.0",
```

Comment:
TBH I was expecting that the babel transforms would handle this for us? as all of our code base gets re-compiled ?why would we need to add a runtime env?

Reply:
Babel only transforms syntax, when it comes to global API that the browser/node supply, you need to have polyfill at runtime.

Reply:
babel-polyfill is already a devDependency (with an older version). Is that now redundant?

Reply:
Good catch @ekohl. Yes this should be a production package so the dev package should be removed. I'll update now.